sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/administration/ServiceBusAdministrationAsyncClientIntegrationTest.java

```diff
@@ -93,23 +91,42 @@ static Stream<Arguments> createHttpClients() {
     @ParameterizedTest
     @MethodSource("createHttpClients")
     void azureIdentityCredentials(HttpClient httpClient) {
-        assumeTrue(interceptorManager.isLiveMode(), "Azure Identity test is for live test only");
+
         final String fullyQualifiedDomainName = TestUtils.getFullyQualifiedDomainName();
+        final TokenCredential tokenCredential = getTokenCredential();
+
+        final ServiceBusAdministrationClientBuilder builder = new ServiceBusAdministrationClientBuilder();
+
+        if (interceptorManager.isPlaybackMode()) {
+            builder.httpClient(interceptorManager.getPlaybackClient());
+        } else if (interceptorManager.isLiveMode()) {
+            builder.httpClient(httpClient)
+                .addPolicy(new RetryPolicy());
```

Comment:
why do we need to add a retry policy in tests? We should have one added by default.

Reply:
Not sure. It was there before for our HTTP client tests. I can remove it.